File: whitespace_gen.py
f = open('whitespace_code_file', 'w')
message = raw_input("Write your message here: \n")
code = ''
for char in message:
	code += '   '
	for num in bin(ord(char))[2:]:
		if num=='1':
			code += '\t'
		elif num=='0':
			code += ' '
	code += '\n'
	code += '\t\n  '
code += '\n\n\n'
f.write(code)
print code

File: whitespace_gen.c
#include<stdio.h>    	    		
#include<string.h>	
int main()     		 		  
{	
FILE *fp; char message[100]; char whitespace[1600]=""; 		  	 	
	
printf("Write your message here (max 99 characters):\n");			 		 
fgets(message,100,stdin);	
for(int i=0;message[i]!=0;i++)     		  	 	
{	
strcat(whitespace,"   ");   			  	 
	
for(int j=7;j>-1;--j)      	     
{	
      		  			
if(message[i]&(1<<j))strcat(whitespace,"\t");	
else strcat(whitespace," ");    		 	  	
}	
strcat(whitespace,"\n\t\n  ");    			  	 
	
}      		 		  
	
strcat(whitespace,"\n\n\n");       	    	
fp=fopen("whitespace_code_file","w");	
fputs(whitespace,fp);         	 	 
fclose(fp);	
return 0; 


}
